new-charts/components/slider-filter.scss

```diff
@@ -13,3 +13,12 @@
 .tc-slider-vue__input::-webkit-slider-runnable-track {
   background: $controllerColor;
 }
+
+.tc-slider-vue__input::-moz-range-track {
+  background: $controllerColor;
+}
+
+.tc-slider-vue__input::-ms-fill-lower,
```

Comment:
Why did you not group the two declaration into one ?

Reply:
It can be done without any problem :)! As it wasn't done on previous declarations, i keeped the old behaviour. But i prefer to stack it too!